src/Input/Select/Select.tsx

```diff
@@ -197,6 +197,14 @@ class Select extends React.Component<Props, State> {
     } else if (e.keyCode === 13) {
       inputElement.blur();
       const activeElement = this.getActiveElement();
+      const activeElementIndex = Number(get(activeElement, 'dataset.id'));
+      const activeChild =
+        isInteger(activeElementIndex) &&
```

Comment:
is `isInteger` necessary here? since `dataset.id` is assigned by `index` which must be an integer.

Reply:
Nope not necessary, added it to be more defensive in case `dataset.id`` gets overwritten somehow.